commit 9363a4a1e121344ae98f22b55fb28623cf0f6038
Author: Anders Ladegaard Marchsteiner <anders.marchsteiner@kghcustoms.com>
Date:   Sun Aug 16 20:26:50 2015 +0200

    Kim'ie has moved Pyonyang half an hour. Not to say several centuries back in time...

diff --git a/panels/datetime/Makefile.am b/panels/datetime/Makefile.am
index 3bf0cf0..3f284ee 100644
--- a/panels/datetime/Makefile.am
+++ b/panels/datetime/Makefile.am
@@ -93,6 +93,10 @@ dist_ui_DATA = \
 	data/timezone_-8_dim.png \
 	data/timezone_8.png \
 	data/timezone_8_dim.png \
+	data/timezone_-8.5.png \
+	data/timezone_-8.5_dim.png \
+	data/timezone_8.5.png \
+	data/timezone_8.5_dim.png \
 	data/timezone_8.75.png \
 	data/timezone_8.75_dim.png \
 	data/timezone_-9.png \
diff --git a/panels/datetime/data/timezone_-8.5.png b/panels/datetime/data/timezone_-8.5.png
new file mode 100644
index 0000000..bdad7bf
Binary files /dev/null and b/panels/datetime/data/timezone_-8.5.png differ
diff --git a/panels/datetime/data/timezone_-8.5_dim.png b/panels/datetime/data/timezone_-8.5_dim.png
new file mode 100644
index 0000000..7caeec7
Binary files /dev/null and b/panels/datetime/data/timezone_-8.5_dim.png differ
diff --git a/panels/datetime/data/timezone_8.5.png b/panels/datetime/data/timezone_8.5.png
new file mode 100644
index 0000000..d210222
Binary files /dev/null and b/panels/datetime/data/timezone_8.5.png differ
diff --git a/panels/datetime/data/timezone_8.5_dim.png b/panels/datetime/data/timezone_8.5_dim.png
new file mode 100644
index 0000000..9612511
Binary files /dev/null and b/panels/datetime/data/timezone_8.5_dim.png differ
